pipeline-scripts/buildlib.groovy

```diff
@@ -125,6 +125,15 @@ def initialize_openshift_dir() {
     echo "Initialized env.OPENSHIFT_DIR: ${env.OPENSHIFT_DIR}"
 }
 
+def init_artcd_working_dir() {
+    sh """
+    mkdir /tmp/artcd_working
+    rsync -a --delete /tmp/artcd_working ./artcd_working
```

Comment:
Can we include a brief comment here, since it is a roundabout way to delete files, for someone who is not familiar with how this is working (me from a few minutes ago :D)

Reply:
Agreed, updated